backend/src/dao/WorkspaceDAO.class.php

```diff
@@ -174,11 +174,17 @@ public function storeFileMeta(int $workspaceId, File $file): void {
     }
 
 
+    public function getFileNames(int $workspaceId): array {
+
+        return $this->_('select type, name from files where workspace_id = ?', [$workspaceId], true);
```

Comment:
Ist es nicht gute SQL-Paxis die Schlüsselwörter in allcaps zu schreiben?
Also in dem Fall: SELECT type, name FROM files WHERE workspace_id = ?